sstables/sstable_set.cc

```diff
@@ -165,6 +165,10 @@ void sstable_set::for_each_sstable(std::function<void(const shared_sstable&)> fu
     return _impl->for_each_sstable(std::move(func));
 }
 
+stop_iteration sstable_set::for_each_sstable_until(std::function<stop_iteration(const shared_sstable&)> func) const {
+    return _impl->for_each_sstable_until(std::move(func));
+}
+
```

Comment:
Wish we could avoid having two interfaces for iterating through all entries. Perhaps we can make for_each_sstable_until() the only in sstable_set_impl, and existing for_each_sstable() can be only in sstable_set and call for_each_sstable_until() with a noop filter.

Reply:
makes sense

Reply:
done in v2